app/assets/stylesheets/application.scss

```diff
@@ -26,6 +26,21 @@
   border-bottom: 1px solid gray;
 }
 
+.card {
+  background-color: #f2580b;
```

Comment:
Color literals like `#f2580b` should only be used in variable declarations; they should be referred to via variable everywhere else.